### ISSUE TITLE
[Feature] Choose disabling each thing 

### ISSUE THREAD

**Issue by ChapChap** [Nepeta/BegoneCIA#8]
Hi,

Thanks you for the app.
May you add the ability to enable/disable each one of either camera, gps or microphone.

Thanks

**Comment by szweee** [comment]
Yes, I hope so too. I also want to have a feature to turn off access to the album.
